subprojects/internal-testing/src/main/groovy/org/gradle/integtests/fixtures/DefaultTestExecutionResult.groovy

```diff
@@ -62,6 +62,17 @@ class DefaultTestExecutionResult implements TestExecutionResult {
         return firstResult
     }
 
+    // In JUnit 3/4 test case name is exactly the method name
+    // In JUnit 5 test method injection is allowed: http://junit.org/junit5/docs/current/user-guide/#writing-tests-dependency-injection
+    // So test case name is [method name + parameters]
+    static String removeParentheses(String testName) {
```

Comment:
Intention of this method is to make legacy tests work for both JUnit 4 and JUnit 5. For example, we have lots of integration tests with something like:

```
given: run 'test'
expect: testClass('SomeClass').assertTestPassed('testMethodName')
```

With `removeParentheses()`, `testMethodName` and `testMethodName()` will be treated equally.